python/llm/src/bigdl/llm/transformers/model.py

```diff
@@ -78,23 +79,23 @@ def from_pretrained(cls,
             if "pretraining_tp" in config_dict:
                 kwargs["pretraining_tp"] = 1
             q_k = load_in_low_bit if load_in_low_bit else "sym_int4"
-            model = cls.load_convert(q_k, *args, **kwargs)
+            model = cls.load_convert(q_k, use_optimize, *args, **kwargs)
         else:
             # load default
             model = cls.HF_Model.from_pretrained(*args, **kwargs)
 
         return model
 
     @classmethod
-    def load_convert(cls, q_k, *args, **kwargs):
+    def load_convert(cls, q_k, use_optimize, *args, **kwargs):
         from .convert import ggml_convert_quant
         invalidInputError(q_k in ggml_tensor_qtype,
                           f"Unknown load_in_low_bit value: {q_k}, expected:"
                           f" sym_int4, asym_int4, sym_int5, asym_int5 or sym_int8.")
         qtype = ggml_tensor_qtype[q_k]
         model = cls.HF_Model.from_pretrained(*args, **kwargs)
         model = model.to("cpu")
-        model = ggml_convert_quant(model, qtype)
+        model = ggml_convert_quant(model, qtype, use_optimize)
```

Comment:
also need to deal with the `ggml_convert_quant` call in `load_low_bit`

Reply:
done